ansible/roles/ocp4-workload-authentication/OWNERS

```diff
@@ -0,0 +1,4 @@
+Red Hat GPTE
+
+Author(s):
+- Wolfgang Kulhanek  (wkulhane@redhat.com)
```

Comment:
That's a good idea.

Can we use a format like yaml ?

What about the standard ansible `meta/main.yml` ? https://galaxy.ansible.com/docs/contributing/creating_role.html#role-metadata

We usually don't target galaxy, but we could take inspiration from it ?

Reply:
Yeah. I'll just create a meta/main in these roles instead of OWNER.